lib/streamlit/caching/memo_decorator.py

```diff
@@ -333,10 +333,16 @@ def __call__(
                 f"Unsupported persist option '{persist}'. Valid values are 'disk' or None."
             )
 
-        # Support passing the params via function decorator, e.g.
```

Comment:
We should update the ttl docstring as well:

```
ttl : float or None
    The maximum number of seconds to keep an entry in the cache, or
    None if cache entries should not expire. The default is None. Note that
    ttl is incompatible with `persist="disk"` - `ttl` will be ignored if
    `persist` is specified.

Reply:
+1